video/out/gpu/d3d11_helpers.c

```diff
@@ -510,7 +511,14 @@ bool mp_d3d11_create_present_device(struct mp_log *log,
         max_fl = max_fl ? max_fl : D3D_FEATURE_LEVEL_11_0;
         min_fl = min_fl ? min_fl : D3D_FEATURE_LEVEL_9_1;
 
-        hr = create_device(log, adapter, warp, opts->debug, max_fl, min_fl, &dev);
+        hr = create_device(log, adapter, warp, debug, max_fl, min_fl, &dev);
+
+        // Retry without debug, if SDK is not available
+        if (debug && hr == DXGI_ERROR_SDK_COMPONENT_MISSING) {
+            debug = false;
```

Comment:
Add an `MP_WARN` explaining that debugging was disabled due to debug SDK not being installed

Reply:
Heh, good idea. Was just doing that. I though create_device will print something, but here there is not fancy `D3D` macro like in placebo.

Reply:
Are you ok with 

```
[vo/gpu-next/d3d11] gpu-debug disabled due to error: The application requested an operation that depends on an SDK component that is missing or mismatched. (0x887a002d)
```

Or should I make it shorter, like

```
Debug layers not available, removing debug flag
```

Reply:
Seems fine to me as-is